react-app/package.json

```diff
@@ -25,7 +25,10 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --watchAll=false --collectCoverage=true --resetMocks=false",
```

Comment:
Had to add the `resetMocks` flag in because I was running into issues with Plantronics where values weren't getting updated even if I forced them to change.  Folks online said it had something to do with `react-scripts` v4.0.3 which we are currently on.